.travis.yml

```diff
@@ -7,7 +7,12 @@ language: c++
 # TODO(2019-07-21): Add "ffmpeg=1" if FFmpeg 4.x becomes available in Ubuntu
 env:
   global:
-    - COMMON_FLAGS="-j4 test=1 mad=1 faad=1 opus=1 modplug=1 wv=1 hss1394=0 virtualize=0 debug_assertions_fatal=1 verbose=0"
+    # For SCons builds
+    - SCONSFLAGS="test=1 mad=1 faad=1 opus=1 modplug=1 wv=1 hss1394=0 virtualize=0 debug_assertions_fatal=1 verbose=0"
+    # For CMake builds
+    - CMAKEFLAGS="-DMAD=ON -DFAAD=ON -DOPUS=ON -DMODPLUG=ON -DWAVPACK=ON -DHSS1394=OFF"
```

Comment:
FAAD2 should be disabled for macOS builds, both for SCons and CMake. I see errors to load the corresponding dynamic library at runtime during the tests. We could move the corresponding options into the platform-dependent EXTRA flags.

Reply:
But we install faad2 via homebrew, strange.